tools/sgen/input/FluentUIStyle.yml

```diff
@@ -288,6 +291,10 @@ MSFPrimaryHeaderTokens extends MSFHeaderFooterTokens:
 AP_MSFDrawerTokens:
   backgroundClearColor: NamedColor(clear)
```

Comment:
I think this token became obsolete (with savings on binary size in mind). If any color other than clear is needed, I believe the dimmed color style should be the used one. Thoughts?

Reply:
I am not sure if I understand how will dimmed color be used to replace this?

Reply:
It'll be the same clear color, just not defined as a token.